src/widget/contentlayout.cpp

```diff
@@ -71,15 +71,11 @@ void ContentLayout::clear()
     QLayoutItem* item;
     while ((item = mainHead->layout()->takeAt(0)) != 0)
     {
-        item->widget()->hide();
-        item->widget()->setParent(nullptr);
```

Comment:
Actually I don't care much, because ContentLayout will vanish anyway after this is done. Anyway, it works here (Qt 5.7) - and if it doesn't, something else is wrong, because:

The `delete item` (the contained widget actually) removes the widget from it's parent's children list and an additional hide is not required. This example is actually given in Qt's docs.